cmake/projects/OpenSSL/schemes/url_sha1_openssl_macos.cmake.in

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+# Scheme for download and install OpenSSL macOS universal library
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include(hunter_dump_cmake_flags)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+include(hunter_unsetvar)
+include(hunter_user_error)
+
+hunter_status_debug("Scheme: url_sha1_openssl_macos")
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_LICENSE_DIR@")
+hunter_test_string_not_empty("@HUNTER_GLOBAL_SCRIPT_DIR@")
+hunter_test_string_not_empty("@HUNTER_TLS_VERIFY@")
+
+# Supported macOS architectures:
+# i386, x86_64, ppc, ppc64
+set(configure_command "./Configure")
+set(configure_architectures @CMAKE_OSX_ARCHITECTURES@)
+
+if(NOT configure_architectures)
+  hunter_status_debug("Using default macOS architecture: x86_64")
+  set(configure_architectures "x86_64")
+endif()
+
+# OpenSSL appends the version number differently by version
+if("@HUNTER_OpenSSL_VERSION@" VERSION_LESS "1.0")
```

Comment:
This block will change on each version update. Is it possible to introduce more stable regular expression?

Reply:
It seems pretty inconsistent, especially since they've just shortened it to `1.1` rather than `1.1.0`. There's no way to know what the next version will be named.
Maybe I can get the correct filename from the symlink.

Reply:
After looking into it a bit more, it looks like OpenSSL 1.1.1 will have the .1.1.dylib suffix, so it looks like they want to keep just the major version numbers in the filename from now on.